src/app/Library/CrudPanel/Traits/Query.php

```diff
@@ -143,9 +143,16 @@ public function count()
         return $this->query->count();
     }
 
+    /**
+     * Apply table prefix in the order clause if the query contains JOINS clauses.
+     *
+     * @param string $column_name
+     * @param string $column_direction
+     * @return \Illuminate\Database\Eloquent\Builder
+     */
     public function orderByWithPrefix($column_name, $column_direction = 'ASC')
     {
-        if ($this->driverIsSql()) {
+        if (null !== $this->query->getQuery()->joins) {
```

Comment:
```suggestion
        if ($this->query->getQuery()->joins !== null) {
```

Reply:
Wouldn't this be more readable? At least that's the way I read stuff.

I've made the suggestion and will commit directly, in the hopes that this time the online tests will run - no idea why they didn't the first time.